frontend/app/settings/page.tsx

```
@@ -28,51 +35,222 @@ const DetailsSection = () => {
+        <div className="flex items-center justify-between rounded-lg border border-gray-200 p-4">
+          <div className="flex items-center gap-3">
+            {currentCompany?.logo_url && (
+              <div className="relative h-8 w-8">
+                <Image
+                  src={currentCompany.logo_url || defaultCompanyLogo}
+                  alt="Workspace Logo"
+                  fill
+                  className="rounded"
+                />
+              </div>
+            )}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)